cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
```

Comment:
I have no objection to that. In my previous PR, I put them in an object CwlEncoders and was asked to put them in the package instead. Where should they go?